sklearn/multiclass.py

```diff
@@ -1025,6 +1025,31 @@ def fit(self, X, y):
 
         return self
 
+    def predict_proba(self, X):
+        """Return faux probability estimates.
+
+        Estimates are based upon euclidean distance between the underlying
+        estimators' predictions and the class code.
+
+        Parameters
+        ----------
+        X : (sparse) array-like of shape (n_samples, n_features)
```

Comment:
```suggestion
        X : {array-like, sparse matrix} of shape (n_samples, n_features)
```

Reply:
That does not match the style of the rest of the unit. Perhaps that should be a separate PR?

Reply:
Yes, the proposed style is the one that we chose. We might have forgotten to edit in some of the places. We can open another PR to solve this.